rpcserver.go

```diff
@@ -1259,9 +1262,9 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 	}
 
 	rpcsLog.Infof("[sendcoins] addr=%v, amt=%v, sat/kw=%v, min_confs=%v, "+
-		"send_all=%v",
+		"send_all=%v, select_outpoints=%v",
```

Comment:
nit: typo, selected_outpoints

Reply:
select can actually be used as an adjective.